packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -305,7 +305,14 @@ function FontLibraryProvider( { children } ) {
 						__( 'There were some errors installing fonts. %s' ),
 						installationErrors.reduce(
 							( errorMessageCollection, error ) => {
-								return `${ errorMessageCollection } ${ error.message }`;
+								if (
```

Comment:
I think it would be good to display them as a list if there remain multiple errors after this. I'm not sure if it's native yet or would require lodash uniq.

Reply:
I have formatted multiple errors as a list and changed the language for single error types.  Updated description to show expected output styling.